examples/suspense/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "suspense"
+version = "0.1.0"
+edition = "2018"
+license = "MIT OR Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+yew = { path = "../../packages/yew" }
+gloo-timers = { version = "0.2.1", features = ["futures"] }
+wasm-bindgen-futures = "0.4"
+wasm-bindgen = "0.2"
+log = "0.4.14"
+console_log = { version = "0.2.0", features = ["color"] }
```

Comment:
Why not use `gloo-console`? It is used everywhere else so we could stick with that for consistency

Reply:
Because log crate is the de-facto standard...

Supports format!(...) style formatting, suppressing logs under certain log levels / prefix, custom log format, etc...

And initialising log gives you output of every crate that uses log.